quartz/components/scripts/explorer.inline.ts

```diff
@@ -160,3 +183,16 @@ function toggleCollapsedByPath(array: FolderState[], path: string) {
     entry.collapsed = !entry.collapsed
   }
 }
+
+/**
+ * Generates hash from string
+ * @param input string to hash
+ * @returns hashed string
+ */
+async function hashString(input: string) {
+  const utf8 = new TextEncoder().encode(input)
+  const hashBuffer = await crypto.subtle.digest("SHA-256", utf8)
+  const hashArray = Array.from(new Uint8Array(hashBuffer))
+  const hashHex = hashArray.map((bytes) => bytes.toString(16).padStart(2, "0")).join("")
+  return hashHex
+}
```

Comment:
```suggestion
function hashString(input: string) {
  return crypto.createHash('sha1').update(input).digest('base64')
}
```

we don't need cryptographic security here, let's use something cheaper

Reply:
Can't seem to get this to work like this, I think this is trying to use the nodejs `crypto` module, which is not available in the frontend/inline script. I can switch to SHA1 with the built in crypto.subtle module, but it would still need to stay async that way

Reply:
ah forgot about that lol, yes let's just switch to SHA1